Demo/src/main/java/com/paypal/android/ui/card/CardFragment.kt

```diff
@@ -120,9 +120,8 @@ class CardFragment : Fragment() {
     }
 
     private suspend fun createOrder() {
-
-        val accessToken = sdkSampleServerAPI.fetchAccessToken().value
-        val configuration = CoreConfig(accessToken = accessToken)
+        val clientId = "AcXwOk3dof7NCNcriyS8RVh5q39ozvdWUF9oHPrWqfyrDS4AwVdKe32Axuk2ADo6rI_31Vv6MGgOyzRt"
```

Comment:
Should we extract this somewhere in the demo app so we don't need to define it in each fragment? I could see a case where we want to change this and need to change it in every fragment.

Reply:
Yeah definitely. There's a sample merchant server PR that needs to get merged. Once it's in, we can switch to fetching it from the merchant server.

Reply:
Is this what we expect merchants to do? My understanding of client IDs was that they are similar to tokenization keys and can live on the client. Maybe I am misunderstanding their intent/use but I assumed we could do something similar for client ID like we do today with tokenization keys on the demo apps in Braintree.

Reply:
Yeh - client IDs can be stored on the client.

I'm not sure what is more common for merchants to do though (store on client or fetch via server). I am guessing fetch via server since I know they can swap client IDs for various products (like Old Navy v Gap v Banana Republic).

We should check with the solutions folks or NXO though to see which is more common.

Reply:
In BT Android we store tokenization keys in [Settings.java](https://github.com/braintree/braintree_android/blob/master/Demo/src/main/java/com/braintreepayments/demo/Settings.java).

Embedding `clientId` as a `BuildConfig` variable works well for merchants with a single set of credentials. For the demo app we may have multiple client ids to test different e2e flows.

Reply:
It'd be nice to not have this value stored in git though. The sample merchant server `GET /client_id` endpoint is the quickest route.